libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -579,17 +562,6 @@ export class PaperHandlerDriver implements PaperHandlerDriverInterface {
         (luminance << 24) | (luminance << 16) | (luminance << 8) | 255;
     }
     await writeImageData(pathOut, colorResult);
-
-    const imageMetadata: ImageFromScanner = {
```

Comment:
Looked like this return isn't currently getting used anywhere, so removed it to simplify both here and in the mock - LMK if it's something we actually need to keep around.